packages/protocol/contracts/thirdparty/README.md

```diff
@@ -4,4 +4,6 @@
 
 - /solmate: code copied from https://github.com/transmissions11/solmate/blob/v7/src/utils/FixedPointMathLib.sol as-is with only solidity pragma changed.
 
-- /nomad-xyz: code copied from https://github.com/nomad-xyz/ExcessivelySafeCall/blob/main/src/ExcessivelySafeCall.sol with unused coded removed and solidity pragma changed.
\ No newline at end of file
+- /nomad-xyz: code copied from https://github.com/nomad-xyz/ExcessivelySafeCall/blob/main/src/ExcessivelySafeCall.sol with unused coded removed and solidity pragma changed.
+
+- /risczero: interface copied from https://sepolia.etherscan.io/address/0x83c2e9cd64b2a16d3908e94c7654f3864212e2f8#code as per: https://dev.risczero.com/api/bonsai/bonsai-on-eth#verifier-contract
```

Comment:
shall we use shall we copy the code from https://github.com/risc0/risc0-ethereum/blob/release-0.7/contracts/src/IRiscZeroVerifier.sol as the reference instead of the deployed contract on a testnet?

Reply:
This one is an interface to the actual contract, which is this (currently):
https://sepolia.etherscan.io/address/0x83c2e9cd64b2a16d3908e94c7654f3864212e2f8#code

We can copy over the interface (this is what we did here), but shall we deploy (exact copy) our own version of receipt verifier implementation ? I dont think so, but i leave it to a decision by you and @Brechtpd

Reply:
As long as we can test the proof locally, I think we should be good. But I guess to do that, we would need to have it somewhere as part of the testing code? On testnet/mainnet yeah just use the one that is already deployed would be the way.